test/support/channel_case.ex

```diff
@@ -20,11 +20,12 @@ defmodule CodeCorpsWeb.ChannelCase do
       # Import conveniences for testing with channels
       use Phoenix.ChannelTest
 
-      alias CodeCorps.Repo
-      import Ecto
-      import Ecto.Changeset
+      import CodeCorps.Factories
+      import CodeCorpsWeb.ChannelCase
```

Comment:
Is there a reason this module is being imported into its own `using`? I'm guessing it's because it's importing its own namespace into the tests, but is that necessary if the case is used?

Reply:
It's the general pattern for other `XCase` modules. Allows us to declare helper methods within this module's namespace, which the test cases can then use.